index.js

```diff
@@ -1,43 +1,20 @@
-// utils
-function $(selector) {
-  return document.getElementById(selector);
-}
-
-const $taskInput = $('#taskInput');
-const $taskList = $('#taskList');
-
-const handleKeydownTaskInput = (event) => {
-  const taskText = $taskInput.value.trim();
-
-  if (event.key !== 'Enter' || !taskText) {
-    return;
-  }
-
-  const taskItem = createTaskItemElement(taskText);
-  $taskList.appendChild(taskItem);
-  $taskInput.value = '';
-};
-
-$taskInput.addEventListener('keydown', handleKeydownTaskInput);
-
-const createTaskItemElement = (text) => {
-  const taskItemLi = document.createElement('li');
-  const taskTextSpan = document.createElement('span');
-  const deleteButtonImg = document.createElement('img');
-  const svgImage =
-    '<svg fill="#af5b5e" width="40px" height="40px" viewBox="-3.5 0 19 19" xmlns="http://www.w3.org/2000/svg" class="cf-icon-svg"><g id="SVGRepo_bgCarrier" stroke-width="0"></g><g id="SVGRepo_tracerCarrier" stroke-linecap="round" stroke-linejoin="round"></g><g id="SVGRepo_iconCarrier"><path d="M11.383 13.644A1.03 1.03 0 0 1 9.928 15.1L6 11.172 2.072 15.1a1.03 1.03 0 1 1-1.455-1.456l3.928-3.928L.617 5.79a1.03 1.03 0 1 1 1.455-1.456L6 8.261l3.928-3.928a1.03 1.03 0 0 1 1.455 1.456L7.455 9.716z"></path></g></svg>';
-
-  taskTextSpan.textContent = text;
-  deleteButtonImg.src = 'data:image/svg+xml,' + encodeURIComponent(svgImage);
-
-  const handleClickDeleteButtonImg = () => {
-    $taskList.removeChild(taskItemLi);
-  };
-
-  deleteButtonImg.addEventListener('click', handleClickDeleteButtonImg);
-
-  taskItemLi.appendChild(taskTextSpan).classList.add('add');
-  taskItemLi.appendChild(deleteButtonImg).classList.add('deleteBtn');
-
-  return taskItemLi;
-};
+import { $ } from './src/utils/dom.js';
+import { todos } from './src/models/todos.js'
+import { addTodo, renderList } from './src/components/todo.js'
+
+// input dom
+const $todoInput = $('#todoInput');
+// 필터 기능 테스트하려고 만든 임의의 버튼
+const $allTodo = $('#allTodo');
+const $leftTodo = $('#leftTodo');
+const $completedTodo = $('#completedTodo');
+const $completedAll = $('#completedAll');
```

Comment:
p4. 실제로 각 html element를 살펴보면 모두 버튼을 가리키고 있는데요~
`$todoInput`과 동일하게 해당 변수가 담고 있는 element인 `button`을 변수명의 마지막에 붙여주면 더 좋을 것 같아요~

변수명은 더 알맞게 지어주면 좋아요~

```suggestion
const $showAllButton = $('#showAllButton');
const $showIncompletedButton = $('#showIncompletedButton');
const $showCompletedButton = $('#showCompletedButton');
const $allCompleteButton = $('#allCompleteButton');
```